packages/main/src/StandardListItem.js

```diff
@@ -75,7 +75,7 @@ const metadata = {
 		 * Defines the state of the <code>info</code>.
 		 * <br>
 		 * Available options are: <code>"None"</code> (by default), <code>"Success"</code>, <code>"Warning"</code> and <code>"Erorr"</code>.
-		 * @type {string}
+		 * @type {ValueState}
```

Comment:
can you also add the @ defaultvalue None

Reply:
done